dash/backend/src/modules/policy/entities/compliance-result-map.ts

```diff
@@ -1,47 +1,73 @@
+/**
+ * Data returned when requesting compliance results for an issue.
+ * Note that complianceReason and severity will only be set if initially
+ * provided at the level (policy, scanner or issue) that determined this issue's compliance
+ */
 interface complianceResult {
-  complianceReason?: string
   compliant: boolean
+  complianceReason?: string
   severity?: string
 }
 
+/**
+ * Interface for a flattened issue from the map.
+ */
 export interface ComplianceIssue {
   policyId: number;
   scannerId: number;
   cve: string;
   compliant: boolean;
   severity?: string;
+  reason?: string;
```

Comment:
Note: this fields (and including it in getAllIssues() is actually a change. This does not affect any current places using this fuctionality.

It made sense to include since the reason being passed in for issues could be potentially useful in future improvements utilizing this function.